### PR TITLE
Add temperature to reports

### PR DESCRIPTION
**Pull request by pgazda**
Add report for temperature readings.
I basically just copied existing weight report. Only change worth mentioning is graph time range selectors. My reasoning is that parent is probably interested in temperature changes over past couple of hours or days (when the child might be sick) instead of past couple of weeks. That's why I used 12h, 24h, 48h, 3d and 7d.

### CODE REVIEW

File: reports/graphs/temperature_change.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    :param objects: a QuerySet of Temperature instances.
+    :returns: a tuple of the graph's html and javascript.
+    """
+    objects = objects.order_by("-time")
+
+    trace = go.Scatter(
+        name=_("Temperature"),
+        x=list(objects.values_list("time", flat=True)),
+        y=list(objects.values_list("temperature", flat=True)),
+        fill="tozeroy",
```
Comment (cdubz) [inline reports/graphs/temperature_change.py:22]: One minor thing here -- lets not use `fill` for this graph in particular. I think the presentation is kind of clearer without it (i.e., it makes it easier to see drastic changes or trends for the temperature).

**Review by cdubz — CHANGES_REQUESTED** [review]
Looks good -- I have just one small CR to remove the fill for the graph. Thanks!

File: reports/graphs/temperature_change.py (hunk 1 above)
Comment (pgazda) [reply reports/graphs/temperature_change.py:22]: Yeah that makes sense.

**Review by cdubz — APPROVED** [review]
Looks good -- thanks!

**Pull request merged**